.github/workflows/stale_issues.yml

```diff
@@ -1,12 +1,11 @@
-name: Mark stale issues and pull requests
+name: (TODO) Mark stale issues and pull requests
 
 on:
-  schedule:
-  - cron: "0 0 * * *"
+#  schedule:
+#  - cron: "0 0 * * *"
```

Comment:
This one theoretically could be used now, but figured we should maybe talk about if we actually want this